flex/CMakeLists.txt

```diff
@@ -121,12 +121,20 @@ find_package(Boost REQUIRED COMPONENTS system filesystem
              context program_options regex thread date_time)
 add_definitions("-DBOOST_BIND_GLOBAL_PLACEHOLDERS")
 
+#find hiactor----------------------------------------------------------------------
```

Comment:
Move the import of `Hiactor` from `bin/CMakeLists.txt` to `CMakeLists.txt`. Otherwise `cmake .. -DBUILD_HQPS=OFF` will fail.  Will investigate this issue later: https://github.com/alibaba/GraphScope/issues/3903